app/views/GovernmentSpendingView.scala.html

```diff
@@ -72,32 +72,35 @@
                 @p(Text(messages("ats.treasury_spending.section.lede")))
             }
 
-            <dl id="tax-spent" class="govuk-summary-list">
-                <div id="tax-spent-header" class="govuk-summary-list__row--no-actions govuk-heading-m">
-                    <dt class="govuk-summary-list__key">@Html(Messages("generic.description"))</dt>
-                    <dd class="govuk-summary-list__value">@Html(Messages("generic.amount_pounds"))</dd>
-                </div>
-
+            <table id="tax-spent" class="govuk-table">
+                <caption class="govuk-visually-hidden">@(Messages("ats.treasury_spending.html.title") + Messages("generic.to_from", viewModel.taxYearFrom, viewModel.taxYearTo))</caption>
```

Comment:
Don't we need a space here between message keys?

Reply:
It's handled in the messages file so don't need to in this case

Reply:
ats.treasury_spending.html.title=Your taxes and public spending
generic.to_from=: {0} to {1}

Reply:
Ah I see. OK.